features/request_construction/basic.feature

```diff
@@ -1,18 +1,16 @@
 Feature: Basic Request Construction
   A simple request with a specified method and path can be sent.
 
-  Backround:
-    Given brine is mixed
-
```

Comment:
this doesn't do anything and will be removed from the rest of the featue files in a subsequent PR. The mixing is done implicitly in the before hook for the test suite and all of the existing steps have typos which make them nops :expressionless: